compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1664,3 +1664,9 @@ error.invalid.readonly.field.type=\
 error.continue.not.allowed=continue not allowed here
 
 error.break.not.allowed=break not allowed here
+
+error.invalid.iterable.type=\
+  invalid iterable type ''{1}'': an iterable object must be subtypes of ''{0}''
+
+error.invalid.iterable.completion.type.in.next.function=\
+  invalid iterable completion type ''{0}'': next method completion type cannot contain type ''{1}''
```

Comment:
This is specifically for `foreach` right? Shall we be more specific in the error? Having error is not an issue for iterable, it is only an issue when used in a foreach statement.